tests/test_uploads_from_komodo.py

```diff
@@ -104,7 +104,13 @@ def test_case_consistency(explorer: Explorer):
     cases = _get_suitable_cases(explorer)
     non_consistent_cases = 0
     for case in cases:
-        res = explorer._sumo.get(f"/admin/consistency-check?case={case.uuid}")
+        mismatches_request = explorer._sumo.get(
+            f"/admin/mismatches?case={case.uuid}"
+        )
+        mismatches_res_location = mismatches_request.headers.get("location")
```

Comment:
maybe just use 
```
res = exp._sumo.poll(mismatches_request)
```
here?

Also, `mismatches_request` is actually an instance of `httpx.Response`.